pages/404.tsx

```diff
@@ -0,0 +1,30 @@
+import { useRouter } from "next/router";
+import { LandingNav, LandingFooter } from "@/Components/components";
+
+const NotFoundPage = () => {
+  const router = useRouter();
+  const handleGoBack = () => {
+    router.back();
+  };
+  return (
+    <div className="flex flex-col h-screen justify-between">
+      <LandingNav pageNotFound={true} />
```

Comment:
```suggestion
      <LandingNav pageNotFound />
```
we can just leave the boolean `true` prop like the above.